src/app/view/components/_common/dot-empty-state/dot-empty-state.component.ts

```diff
@@ -0,0 +1,31 @@
+import { Component, Input, OnInit } from '@angular/core';
+
+@Component({
+    selector: 'dot-empty-state',
+    templateUrl: './dot-empty-state.component.html',
+    styleUrls: ['./dot-empty-state.component.scss']
+})
+export class DotEmptyStateComponent implements OnInit {
+    @Input()
+    rows: number;
+
+    @Input()
+    cols: number[];
+    inputWidth: string;
+
+    constructor() {}
+
+    /**
+     * Return rows for empty template
+     * @param number n
+     * @returns number[]
+     * @memberof DotEmptyStateComponent
+     */
+    numberOfRows(): number[] {
+        return Array(this.rows).fill(0);
+    }
+
+    ngOnInit(): void {
+        this.inputWidth = `${96.5 / this.cols.length}%`; // 100% - width of checkbox = 96.5%, which is then divided by number of cols
```

Comment:
Set the `width of checkbox` as a constant.

Reply:
If I get this right, all the columns are the same width BUT the number we were in the `rows` is for the skeleton text inside the `<td>`? 

if so... is this a good approach? Or should we use the number we pass as the `width` of the columns since in the input `cols` I think that's what we are saying were doing.